composer.json

```diff
@@ -19,10 +19,10 @@
         }
     ],
     "require": {
-        "php": ">=7.3",
+        "php": "^8.1",
         "ext-json": "*",
-        "nesk/rialto": "^1.2.0",
-        "psr/log": "^1.0|^3.0",
+        "nesk/rialto": "dev-zoon",
+        "psr/log": "^3.0",
```

Comment:
Can we keep support of ^1? This is public library, we need it as flexible as possible

Reply:
No, required for "nesk/rialto": "dev-zoon"